openapi.yaml

```diff
@@ -0,0 +1,645 @@
+    url: http://www.gnu.org/licenses/agpl-3.0.en.html
+  contact:
+    name: IS Data Architecture Team
+    url: https://is.oregonstate.edu/data-architecture
+    email: isdataarchitecture@oregonstate.edu
+schemes:
+  - http
+host: 'api.oregonstate.edu'
+basePath: /v1
+externalDocs:
+  description: GitHub Repository
+  url: https://github.com/osu-mist/game-curator-api
+produces:
+  - application/json
+paths:
+  /games:
+    parameters:
+      - $ref: '#/parameters/authorization'
+    get:
+      summary: List all games
+      description: Get all games
+      tags:
+        - Games
+      parameters:
+        - $ref: '#/parameters/pageNumber'
+        - $ref: '#/parameters/pageSize'
+        - name: developerId
+          in: query
+          type: integer
+          required: false
+          description: id of developer to return games from
+        - name: name
+          in: query
+          type: string
+          required: false
+          description: name of game to return
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/GameResults'
+        '400':
+          description: Bad request
+          schema:
+            $ref: '#/definitions/Errors'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/Errors'
+    post:
+      summary: Add a new game
+      tags:
+        - Games
+      parameters:
+        - in: body
+          name: body
+          required: true
+          schema:
+            $ref: '#/definitions/GamePostBody'
+      responses:
+        '201':
+          description: Successfully added a game
+        '400':
+          description: Bad request
+          schema:
+            $ref: '#/definitions/Errors'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/Errors'
+  /games/{gameId}:
+    parameters:
+    - $ref: '#/parameters/authorization'
+    - name: gameId
+      in: path
+      description: Id of game to return
+      required: true
+      type: integer
+      format: int64
+    get:
+      summary: Find game by ID
+      description: Returns a single game
+      tags:
+        - Games
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/GameResult'
+        '404':
+          description: Game not found
+          schema:
+            $ref: '#/definitions/Errors'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/Errors'
+    patch:
+      summary: Update information on a game
+      description: Pass in parameters to update fields on game resources
+      tags:
+        - Games
+      parameters:
+      - name: body
+        in: body
+        required: true
+        schema:
+          $ref: '#/definitions/GameResource'
+      responses:
```

Comment:
You can't update the id of the game in a patch.

Reply:
added GamePatchBody